p2p/p2p_test.go

```diff
@@ -161,6 +160,40 @@ func TestStart(t *testing.T) {
 	}
 }
 
+func TestService_PeerCount(t *testing.T) {
+	ipfsNode, err := StartIpfsNode()
```

Comment:
A lot of whats after this is duplicated from other tests in here. Please refactor or open an issue to ensure it gets done.